pkgs/development/rocm-modules/6/llvm/base.nix

```diff
@@ -86,7 +86,7 @@ in stdenv.mkDerivation (finalAttrs: {
     cmake
     ninja
     git
-    python3Packages.python
+    (python3Packages.python.withPackages (p: [ p.setuptools ]))
```

Comment:
Have you tried this?

```suggestion
    python3Packages.python
    python3Packages.setuptools
```

I ask because in `pkgs/development/rocm-modules/6/llvm/base.nix`, `recommonmark` is included in a similar fashion, and it seems to work there.

Reply:
Already one of the methods tried and failed.

Reply:
Understood, sad times.

Reply:
Yep, made no sense to me...